griffon/services/core_queries.py

```diff
@@ -300,37 +309,129 @@ def execute(self, status=None) -> List[Dict[str, Any]]:
             "include_fields": "purl,type,name,related_url,namespace,software_build,nvr,release,version,arch,product_streams.product_versions,product_streams.name,product_streams.ofuri,product_streams.active,product_streams.exclude_components",  # noqa
         }
         if self.search_latest:
-            params["latest_components_by_streams"] = "True"
             if not self.strict_name_search:
                 params["re_name"] = self.component_name
             else:
                 params["name"] = self.component_name
             if self.ns:
                 params["namespace"] = self.ns
-            status.update("griffoning: searching latest component(s).")
+            if not (self.include_inactive_product_streams):
+                params["active_streams"] = "True"
+            params["root_components"] = "True"
+            params["latest_components_by_streams"] = "True"
+            status.update("griffoning: searching latest root component(s).")
```

Comment:
This could be also somehow wrapped so the status bar text is always prefixed with griffoning. Will include that in my future PRs as well.

Reply:
agreed for future PR